beamer/agent/chain.py

```diff
@@ -12,15 +12,15 @@
 from web3.contract import Contract
 from web3.types import Timestamp, Wei
 
-from beamer.agent.events import Event, EventFetcher
-from beamer.agent.l1_resolution import run_relayer_for_tx
 from beamer.agent.models.claim import Claim
 from beamer.agent.models.request import Request
+from beamer.agent.relayer import run_relayer_for_tx
 from beamer.agent.state_machine import Context, process_event
-from beamer.agent.typing import BlockNumber, ChainId
-from beamer.agent.util import TransactionFailed, load_ERC20_abi, transact
+from beamer.events import Event, EventFetcher
+from beamer.typing import BlockNumber, ChainId
+from beamer.util import TransactionFailed, get_ERC20_abi, transact
 
-_ERC20_ABI = load_ERC20_abi()
+_ERC20_ABI = get_ERC20_abi()
```

Comment:
We can get rid of this global now and instead just call `get_ERC20_abi` where needed.